.travis.yml

```diff
@@ -64,6 +64,8 @@ matrix:
       env: NPY_RELAXED_STRIDES_CHECKING=0
     - python: 3.6
       env: USE_WHEEL=1 NPY_RELAXED_STRIDES_DEBUG=1
+    - python: 3.6
+      env: NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=0
```

Comment:
Should we consolidate this CI run into one of the others?

Reply:
We get 5 parallel runs and we are already over 10, so as long as there are fewer than 15 test runs it shouldn't make much difference. The runs vary in time, so it isn't possible to predict exactly what will happen, but close enough.